### PR TITLE
core-288 allow template to be blank

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

File: src/common_swagger_api/schema/webhooks.clj [hunk 1]
```diff
@@ -15,7 +15,7 @@
 (defschema WebhookType
   {(optional-key :id)       (describe UUID "A UUID for the type")
    :type                    (describe NonBlankString "Webhook type")
-   (optional-key :template) (describe NonBlankString "Template for this Webhook type")})
+   (optional-key :template) (describe "Template for this Webhook type")})
```
Comment (psarando) [inline src/common_swagger_api/schema/webhooks.clj]: This still needs a type to `describe`:
```suggestion
   (optional-key :template) (describe String "Template for this Webhook type")})
```
You can run `lein eastwood` in this repo's directory.
It's a linter that will check for errors without having to build and run this in our actual services first.
If you run it now, it should complain about a syntax error here.

**Review by psarando — CHANGES_REQUESTED** [review]
This will throw a syntax error once it's imported into our services (or maybe even before that when Jenkins tries to build it).

Alternately, we could update the service to not include the `:template` key if it has no value, but allowing an empty string `template` works too.

File: src/common_swagger_api/schema/webhooks.clj (hunk 1 above)
Comment (sri2k1us) [reply src/common_swagger_api/schema/webhooks.clj]: The `template`key is not used by the UI at the moment. So either allowing blank or not including the key all together makes no difference 👍

**Review by psarando — APPROVED** [review]
LGTM :shipit:

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**